### PR TITLE
initial draft of trimmed template

### PR DESCRIPTION
**Pull request by aimalz**
To address #26, I took out most of the checkboxes, leaving just the ones that are most relevant to the RAIL team's workflow.

### CODE REVIEW

**Review by eacharles — APPROVED** [review]
Better, still longer that I think people will actually pay attention to, but we shall see.

**Comment by aimalz** [comment]
@eacharles Yeah, I have to agree. I trimmed out the explanatory text and consolidated the checkboxes further. What do you think?

**Review by OliviaLynn — APPROVED** [review]

**Review by sschmidt23 — APPROVED** [review]
it's a little odd that three of the four have a "positive" check box (i.e. checked  = good), while the last one about including a breaking change is unchecked = good.  Maybe rephrase the last one to be "I do not include any breaking changes" as the default, and thus seeing the box unchecked would stand out more?
Not a big deal, but I did notice it. I'll approve, change the above if you feel like it.
